### PR TITLE
[WEB-49] Basics Basal-IQ auto suspends

### PR DESCRIPTION
**Pull request by clintonium-119**
Part of [WEB-49]

Related PRs:
https://github.com/tidepool-org/tideline/pull/429
https://github.com/tidepool-org/blip/pull/1014

[WEB-49]: https://tidepool.atlassian.net/browse/WEB-49?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/modules/print/BasicsPrintView.js [hunk 1]
```diff
@@ -167,7 +167,7 @@ class BasicsPrintView extends PrintView {
 
     this.renderCalendarSection({
       title: this.sections.basals.title,
-      data: this.aggregationsByDate.basals.byDate,
+      data: this.aggregationsByDate.basals.basal.byDate, // TODO: do we need to also add the suspend deviceEvents here?
```
Comment (krystophv) [inline src/modules/print/BasicsPrintView.js:170]: Has this question been answered somewhere?

**Review by krystophv — APPROVED** [review]
Nice work, LGTM 🚢 

Did have a question about the `TODO`, but not so much to not approve the PR

File: src/modules/print/BasicsPrintView.js (hunk 1 above)
Comment (clintonium-119) [reply src/modules/print/BasicsPrintView.js:170]: Ah, good catch.  I realized the answer is no, we're good with just passing the basal data in here.  Forgot I'd put this in.

Comment (clintonium-119) [reply src/modules/print/BasicsPrintView.js:170]: Will remove the comment and merge

**Pull request merged**